scania-digital-design-system/sdds-website
# bug(links): fix links to contribution page

**Describe pull-request**  
- fixed path to go to contribution page from landing page
- fixed path to go to bug report, feature request, propse a new design from contribution info page

**Solving issue**  
Fixes: #312 #311 

**How to test**  
- click 'let us know' button on landing page(on the bottom of the page) -> check if it opens `/contribution/contribute-info `
- on contribution/contribute-info page, click cards 'report a bug', 'request feature', 'propose a new design'
   -> check if they open report forms accordingly.
  *'propose a new design' card opens a page to choose a template type since we don't have pre-made template for it
